pvacseq/lib/main.py

```diff
@@ -190,6 +196,9 @@ def main(args_input = sys.argv[1:]):
     if args.fasta_size%2 != 0:
         sys.exit("The fasta size needs to be an even number")
 
+    if args.iedb_retries > 100:
+        sys.exit("The number of IEDB retries must be less than or equal to 100")
```

Comment:
I'm open to changing it but I think that we should then change i for all extra argument checking we are doing.